### PR TITLE
Nuget proof

### LINKED ISSUE #517: Nuget install does not work
When trying to install nuget package from repo:

```
C:\Git\artipie\examples\nuget\sample-consumer>nuget install samplefordeployment -Version 1.0.0 -Source http://localhost:8080/my-nuget/index.json
Feeds used:
  http://localhost:8080/my-nuget/index.json



Attempting to gather dependency information for package 'samplefordeployment.1.0.0' with respect to project 'C:\Git\artipie\examples\nuget\sample-consumer', targeting 'Any,Version=v0.0'
An error occurred while retrieving package metadata for 'samplefordeployment.1.0.0' from source 'http://localhost:8080/my-nuget/index.json'.
  Response status code does not indicate success: 500 (Internal Server Error).
```

The stack trace shows that there is conflict between versions of guava.

### PR DESCRIPTION
**Pull request by Sammers21**
Fixes #517 NuGet install 

Related to #154

### CODE REVIEW

**Comment by Sammers21** [comment]
@olegmoz, review the pr please

File: examples/nuget/run.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+nuget locals all -list
+
+# Create a NuGet package.
+nuget pack
+
+# Push the package to Artipie.
+nuget push SampleForDeployment.1.0.0.nupkg -SkipDuplicate -src http://localhost:8080/my-nuget/index.json
+
+# Getting to the consumer dir.
+Set-Location ../sample-consumer
```
[the remark this answers is not in the document]
Comment (Sammers21) [reply examples/nuget/run.ps1:24]: @olegmoz, we have the same name convention across all the proofs.

**Comment by Sammers21** [comment]
@olegmoz, review the pr please

**Review by olegmoz — APPROVED** [review]
@Sammers21 thanks, all is good

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#516` is not assigned, can't get performer

**Pull request merged**